helm/alfresco-content-services-community/values.yaml

```diff
@@ -28,6 +28,21 @@ repository:
       -Dsolr.secureComms=none
       -Dindex.subsystem.name=solr6
       -Ddeployment.method=HELM_CHART
+
+      -Dlocal.transform.service.enabled=true
+      -DlocalTransform.pdfrenderer.url=http://alfresco-pdf-renderer:8090/
```

Comment:
I think these URLs need to have service names in https://github.com/Alfresco/acs-community-deployment/blob/master/helm/alfresco-content-services-community/templates/config-repository.yaml instead of values.yaml

Reply:
They may match at the moment, but those should not be hardcoded here.